src/api-types.ts

```diff
@@ -125,55 +127,62 @@ export const TokenizeInputSchema = z.object({
 
 export type TokenizeInput = z.infer<typeof TokenizeInputSchema>;
 
-export interface TokenizeResult {
-  token_count: number;
-  tokens?: string[];
-}
-
-export interface TokenizeOutput {
-  model_id: string;
-  created_at: string;
-  results: TokenizeResult[];
-}
+export const TokenizeOutputSchema = z.object({
+  model_id: z.string(),
+  created_at: z.string(),
+  results: z.array(
+    z.object({
+      token_count: z.number().int().nonnegative(),
+      tokens: z.array(z.string()),
+    }),
+  ),
+});
+export type TokenizeOutput = z.output<typeof TokenizeOutputSchema>;
 
 // MODELS
 
-export interface ModelsOutput {
-  results: {
-    id: string;
-    name: string;
-    size: string;
-    token_limit: number;
-  }[];
-}
+export const ModelsOutputSchema = z.object({
+  results: z.array(
+    z.object({
+      id: z.string(),
+      name: z.string(),
+      size: z.string(),
+      token_limit: z.number().int().nonnegative(),
+    }),
+  ),
+});
+export type ModelsOutput = z.output<typeof ModelsOutputSchema>;
 
-type ModelSchema = { id: number; value: any };
+const ModelSchemaSchema = z.object({ id: z.number().int(), value: z.any() });
```

Comment:
I think that `id` is `string`, not a `number`

Reply:
I rather checked but it's really an integer.